src/scripts/components/Play.js

```diff
@@ -584,6 +588,12 @@ class Play extends Component<Props, State> {
       } else {
         player.play()
       }
+
+      if (this.state.isStartScreen) {
+        this.setState({
+          isStartScreen: false
```

Comment:
Do we show the start screen basically anytime the video is not playing? If that is the case, then we should just remove `isStartScreen` from state and base whether we show the start screen solely off of `this.props.isPlaying`. This will avoid data duplication.

Do you see something wrong w/ this approach? Even if we can't do it _entirely_ from `this.props.isPlaying` I'd bet we can derive whether to show the start screen from that prop and maybe another.

Reply:
> Do we show the start screen basically anytime the video is not playing?

No. Just in embed. One time only.
I don't understand this approach

Reply:
My point is, why do we have `this.state.isStartScreen`? Can't the logic as to whether we show the screen just be `if (this.props.isEmbed && this.props.isPlaying)`?

Reply:
You could write a method like this:

```javascript
shouldShowStartScreen() {
  return this.props.isEmbed && this.props.isPlaying
}
```

Assuming that is indeed the correct logic

Reply:
The `this.props.isEmbed && this.props.isPlaying` could be a *paused video state* or a *ended video state*. And this *start state* is different from both.

Reply:
@jrgarou I think my concerns here were legitimate. If you visit the `/embed` url with `autoplay=true`, you'll see that the start screen does not go away. I will work on a fix now and tag you in PR